test/integration/builder/selects.js

```diff
@@ -540,6 +540,45 @@ module.exports = function(knex) {
           });
       });
 
+      it.only('allows parameter to be a query builder', function() {
```

Comment:
Good call. Upon writing something similar, I found that this PR would crash on `.orWhere`. It's all fixed up now though.